### ISSUE TITLE
Cannot resolve ktor-auth-jwt

### ISSUE THREAD

**Issue by tristancaron** [ktorio/ktor#277]
The warning:
```
Warning:<i><b>root project 'griffin-api': Unable to build Kotlin project configuration</b>
Details: java.lang.reflect.InvocationTargetException: null
Caused by: org.gradle.api.internal.artifacts.ivyservice.DefaultLenientConfiguration$ArtifactResolveException: Could not resolve all files for configuration ':compileClasspath'.
Caused by: org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find io.ktor:ktor-auth-jwt:0.9.0.
Searched in the following locations:
    https://jcenter.bintray.com/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.pom
    https://jcenter.bintray.com/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.jar
    https://repo1.maven.org/maven2/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.pom
    https://repo1.maven.org/maven2/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.jar
    http://dl.bintray.com/kotlin/ktor/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.pom
    http://dl.bintray.com/kotlin/ktor/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.jar
    https://dl.bintray.com/kotlin/kotlinx/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.pom
    https://dl.bintray.com/kotlin/kotlinx/io/ktor/ktor-auth-jwt/0.9.0/ktor-auth-jwt-0.9.0.jar
Required by:
    project :</i>
```

The gradle:
```
dependencies {
    val kotlinVersion by extra { "1.2.0" }
    val ktorVersion by extra { "0.9.0" }

    compile(kotlin("stdlib", kotlinVersion))

    // ktor
    compile("io.ktor:ktor-server-core:$ktorVersion")
    compile("io.ktor:ktor-server-netty:$ktorVersion")

    // ktor - features
    compile("io.ktor:ktor-auth:$ktorVersion")
    compile("io.ktor:ktor-auth-jwt:$ktorVersion")

    // Logging
    compile("ch.qos.logback:logback-classic:1.2.1")
}
```

**Comment by orangy** [comment]
`jwt` was merged after 0.9.0 was released, you need latest alpha build

**Comment by orangy** [comment]
Please try 0.9.1 and tell us if it works or not for you. Closing for now.
